crates/anvil/src/eth/miner.rs

```diff
@@ -272,3 +281,47 @@ impl fmt::Debug for ReadyTransactionMiner {
             .finish_non_exhaustive()
     }
 }
+
+/// This struct represents mining mode in a simplified, serializable format
+#[derive(Debug, Serialize, PartialEq, Eq)]
+#[serde(rename_all = "camelCase")]
+pub enum SerializableMiningMode {
```

Comment:
```suggestion
pub enum MiningModeKind {
```

Reply:
There is already a MiningMode in the code, not MiningModeKind, so I think SerializableMiningMode is a reasonable name to indicate it’s the serializable version. 

However, thanks for the suggestion! I’ll consider using the kind suffix for future enums.